REST/src/main/java/pl/jakubowskiprzemyslaw/REST/services/JSONService.java

```diff
@@ -0,0 +1,76 @@
+package pl.jakubowskiprzemyslaw.REST.services;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import pl.jakubowskiprzemyslaw.tajgertim.models.board.Mast;
+import pl.jakubowskiprzemyslaw.tajgertim.models.board.Ship;
+import pl.jakubowskiprzemyslaw.tajgertim.models.coordinates.Coordinate;
+import pl.jakubowskiprzemyslaw.tajgertim.models.shoot.ShootResult;
+import pl.jakubowskiprzemyslaw.tajgertim.models.view.BoardsView;
+
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.atomic.AtomicInteger;
+
+@Service
+public class JSONService {
+
+    private final ObjectMapper mapper;
+
+    @Autowired
+    public JSONService(ObjectMapper mapper) {
```

Comment:
self comment: why mapper is here and in the PlayingController?

Reply:
Will be refactored later. 
Tracked under #70